### ISSUE TITLE
ClusterExternalSecret Inactive/Not Working

### ISSUE THREAD

**Issue by cazter** [external-secrets/external-secrets#882]
After creating the ClusterExternalSecret, nothing happens. No change in status to the ClusterExternalSecret. No creation of an ExternalSecret. No creation of a K8S secret. No noticeable response from the controller pod logs.

```yaml
apiVersion: external-secrets.io/v1beta1
kind: ClusterExternalSecret
metadata:
  name: runtime-config
spec:
  externalSecretName: "runtime-config-es"
  refreshTime: 5s
  namespaceSelector:
    matchLabels:
      externalsecrets: enabled
  externalSecretSpec:
    secretStoreRef:
      name: cluster-secretstore-aws
      kind: ClusterSecretStore
    refreshInterval: 5s
    target:
      name: runtime-config
      creationPolicy: Owner
    data:
      - secretKey: COMMON_FREETRIAL_API_KEY
        remoteRef:
          key: common/COMMON_FREETRIAL_API_KEY
```

```shell
$ ku get ClusterExternalSecret
NAME                  AGE
runtime-config   22m
```

```shell
$ ku describe ClusterExternalSecret runtime-config
Name:         runtime-config
Namespace:
Labels:       <none>
Annotations:  <none>
API Version:  external-secrets.io/v1beta1
Kind:         ClusterExternalSecret
Metadata:
  Generation:          1
  Managed Fields:
    API Version:  external-secrets.io/v1beta1
    Fields Type:  FieldsV1
    fieldsV1:
      f:metadata:
        f:annotations:
          .:
          f:kubectl.kubernetes.io/last-applied-configuration:
      f:spec:
        .:
        f:externalSecretName:
        f:externalSecretSpec:
          .:
          f:data:
          f:refreshInterval:
          f:secretStoreRef:
            .:
            f:kind:
            f:name:
          f:target:
            .:
            f:creationPolicy:
            f:deletionPolicy:
            f:name:
        f:namespaceSelector:
          .:
          f:matchLabels:
            .:
            f:es:
        f:refreshTime:
    Manager:         kubectl-client-side-apply
    Operation:       Update
    Time:            2022-03-23T23:04:09Z
  Resource Version:  355091728
  UID:               159c762e-4201-421d-a626-3b8bc7154ad2
Spec:
  External Secret Name:  runtime-config-es
  External Secret Spec:
    Data:
      Remote Ref:
        Conversion Strategy:  Default
        Key:                  common/COMMON_FREETRIAL_API_KEY
      Secret Key:             COMMON_FREETRIAL_API_KEY
    Refresh Interval:         5s
    Secret Store Ref:
      Kind:  ClusterSecretStore
      Name:  cluster-secretstore-aws
    Target:
      Creation Policy:  Owner
      Deletion Policy:  None
      Name:             runtime-config
  Namespace Selector:
    Match Labels:
      externalsecrets:        enabled
  Refresh Time:  5s
Events:          <none>
```

**Comment by cazter** [comment]
Noticed that the controller startup doesn't seem to include reconciler for ClusterExternalSecret. 

```
{"level":"info","ts":1648078007.3545947,"logger":"controller-runtime.metrics","msg":"Metrics server is starting to listen","addr":":8080"}
{"level":"info","ts":1648078007.355407,"logger":"setup","msg":"starting manager"}
{"level":"info","ts":1648078007.3557103,"msg":"Starting server","path":"/metrics","kind":"metrics","addr":"[::]:8080"}
{"level":"info","ts":1648078007.3558464,"logger":"controller.secretstore","msg":"Starting EventSource","reconciler group":"external-secrets.io","reconciler kind":"SecretStore","source":"kind source: *v1alpha1.SecretStore"}
{"level":"info","ts":1648078007.3558762,"logger":"controller.secretstore","msg":"Starting Controller","reconciler group":"external-secrets.io","reconciler kind":"SecretStore"}
{"level":"info","ts":1648078007.3560848,"logger":"controller.externalsecret","msg":"Starting EventSource","reconciler group":"external-secrets.io","reconciler kind":"ExternalSecret","source":"kind source: *v1alpha1.ExternalSecret"}
{"level":"info","ts":1648078007.356121,"logger":"controller.externalsecret","msg":"Starting EventSource","reconciler group":"external-secrets.io","reconciler kind":"ExternalSecret","source":"kind source: *v1.PartialObjectMetadata"}
{"level":"info","ts":1648078007.3561246,"logger":"controller.externalsecret","msg":"Starting Controller","reconciler group":"external-secrets.io","reconciler kind":"ExternalSecret"}
{"level":"info","ts":1648078007.3562586,"logger":"controller.clustersecretstore","msg":"Starting EventSource","reconciler group":"external-secrets.io","reconciler kind":"ClusterSecretStore","source":"kind source: *v1alpha1.ClusterSecretStore"}
{"level":"info","ts":1648078007.3562777,"logger":"controller.clustersecretstore","msg":"Starting Controller","reconciler group":"external-secrets.io","reconciler kind":"ClusterSecretStore"}
{"level":"info","ts":1648078007.456963,"logger":"controller.clustersecretstore","msg":"Starting workers","reconciler group":"external-secrets.io","reconciler kind":"ClusterSecretStore","worker count":1}
{"level":"info","ts":1648078007.456936,"logger":"controller.secretstore","msg":"Starting workers","reconciler group":"external-secrets.io","reconciler kind":"SecretStore","worker count":1}
{"level":"info","ts":1648078007.4569533,"logger":"controller.externalsecret","msg":"Starting workers","reconciler group":"external-secrets.io","reconciler kind":"ExternalSecret","worker count":2}
```
We're using the default helm install with --set concurrent=2. There don't appear to be any associated chart values or arguments examples within the repo for including the missing reconciler to the startup?

**Comment by cazter** [comment]
Uninstalled helm chart and had to manually removed the CRD for clusterexternalsecrets.external-secrets.io. Reinstalled and found that clusterexternalsecret crd is not being included.
```
$ helm -n devops install externalsecrets external-secrets/external-secrets --set concurrent=2 --set installCRDs=true
NAME: externalsecrets
LAST DEPLOYED: Wed Mar 23 17:49:41 2022
NAMESPACE: devops
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
external-secrets has been deployed successfully!

In order to begin using ExternalSecrets, you will need to set up a SecretStore
or ClusterSecretStore resource (for example, by creating a 'vault' SecretStore).

More information on the different types of SecretStores and how to configure them
can be found in our Github: https://github.com/external-secrets/external-secrets
$ ku get crd
NAME                                         CREATED AT
clustersecretstores.external-secrets.io      2022-03-23T23:49:49Z
externalsecrets.external-secrets.io          2022-03-23T23:49:48Z
secretstores.external-secrets.io             2022-03-23T23:49:48Z
```

**Comment by gusfcarvalho** [comment]
Hi @cazter ! That’s because ClusterExternalSecrets has not been released!

if you want to give it a try though, you can manually install the controller from the main branch.

**Comment by gusfcarvalho** [comment]
Expect `ClusterExternalSecrets` to be available in v0.5.0. We are expecting to release it in two weeks. If you have any further questions, feel free to reopen this issue!

**Comment by cazter** [comment]
IF I SEE CRD, I INSTALL. Hah, only kidding. Thanks for clearing this up.
